samples/apps/github-qna-webapp-react/README.md

```diff
@@ -15,7 +15,7 @@
 
 ## About the GitHub Repo Q&A Bot Sample
 
-The GitHub Repo Q&A Bot sample allows you to pull in data from a public GitHub repo into a local memory store in order to ask questions about the project and to get answers about it.  The sample highlights how [memory](TODO Link to memory) and [embeddings](TODO Link to embeddings) work along with X SK Functions when the size of the data is larger than the allowed token limited.  Each SK function will call Open AI to perform the tasks you ask about.​
+The GitHub Repo Q&A Bot sample allows you to pull in data from a public GitHub repo into a local memory store in order to ask questions about the project and to get answers about it. The sample highlights how [memory](TODO Link to memory) and [embeddings](TODO Link to embeddings) work along with X SK Functions when the size of the data is larger than the allowed token limited.  Each SK function will call Open AI to perform the tasks you ask about.​
```

Comment:
along with X SK Functions 

Solve for X?

Reply:
[memory](TODO Link to memory) and [embeddings](TODO Link to embeddings)

Reply:
Good catch! Fixed.